### DISCUSSION TITLE
Showing type name of objects

### DISCUSSION THREAD

**Discussion by nidefawl** [microsoft/vscode-cpptools#8981] [Q&A]
When debugging I frequently need to lookup the type name of polymorphic objects.

The Variables window and watch window do show the contents of pointers, but not the type of the object instance.

As a workaround I can add "this.someMissingField" to the watch window and get an error message containing the type name `this` points to. 

Is there a way to always show the type name of pointers?

**Comment by terriblememory** [comment]
I opened an issue for this problem before I noticed this.

https://github.com/microsoft/vscode-cpptools/issues/11657#issue-1988096556

If you already have an open issue for it let me know.
